src/arguments.cpp

```diff
@@ -104,6 +106,31 @@ Toggle Arguments::asToggle()
     return strcmp(arg, enable) == 0 ? Toggle::enable : Toggle::disable;
 }
 
+const char* Arguments::asNetInterface()
+{
+    const char* arg = asText();
+
+    bool found = true;
```

Comment:
It seems to me here is a logic error.
The `getifaddrs` failure will mean that the interface found.

Reply:
No, it works as I expected - skip check if `getifaddrs` fails. But I think you are right.